labrad/wrappers.py

```diff
@@ -24,9 +24,9 @@
 
 from zope.interface import implements
 
-from labrad import constants as C, manager, protocol
+from labrad import constants as C, manager, protocol, types as T
```

Comment:
At least don't add new "from labrad import X" imports, especially for modules when you can just do a regular import for the same effect.